json2junit/testdata/inputs/chatty.json

```diff
@@ -0,0 +1,100 @@
+{"Time":"2025-02-07T14:43:16.4746636+01:00","Action":"start","Package":"cmd/go"}
+{"Time":"2025-02-07T14:43:16.4751944+01:00","Action":"run","Package":"cmd/go","Test":"TestScript"}
```

Comment:
`.jsonl` will make VS Code's syntax checker happy, by the way. ("JSON Lines".)

Reply:
Didn't know this trick, thanks!